.github/workflows/native-image-snapshot.yml

```diff
@@ -61,14 +61,10 @@ jobs:
         shell: bash
         run: echo "DATETIME=$(date -u +'%F at %T (UTC)')" >> $GITHUB_ENV
 
-      - name: Collect Metadata Automatically with Tracing Agent
-        shell: bash
-        run: ./gradlew :plantuml-gplv2:nativeTest
-
-      - name: Copy Metadata
-        shell: bash
-        run: ./gradlew :plantuml-gplv2:metadataCopy
-
+      - name: Generate reachability metadata with agent (PDF)
+        run: ./gradlew :plantuml-gplv2:run --args='-tpdf -v /home/finkel/work_self/plantuml/samples/' metadataCopy
```

Comment:
Hi,

As mentioned on #2107.

Beware of local machine variable as:
- `/home/finkel/work_self/`

Regards,
Th.